apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -403,7 +404,10 @@ private async Task<PooledSession> BeginTransactionAsync(PooledSession session, T
                 var request = new BeginTransactionRequest { Options = options };
                 try
                 {
-                    var transaction = await session.BeginTransactionAsync(request, Options.Timeout, cancellationToken).ConfigureAwait(false);
+                    var callSettings = Client.Settings.BeginTransactionSettings
```

Comment:
I don't think we need it, to be honest. It would be really, really weird for the default CallSettings for a client to have a cancellation token.
(Whereas having one cancellation token when starting a query and another on a MoveNext call is more reasonable.)